### PR TITLE
Updated business form

### LINKED ISSUE #1790: Updated business form
## Summary

We have added different fields to the business page for victims who are reporting a cybrecrime related to their business. 

## Design detail

We have added different fields to the business page for victims who are reporting a cybrecrime related to their business. 
Iteration 4

What happened

https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-9?node-id=3521%3A6371

## Unresolved questions

This is related to the change of the "What Happened" page, but this has been created as a separate issue following this.

### PR DESCRIPTION
**Pull request by kevinyang06**
Fixes #1790

# Description

> We have added different fields to the business page for victims who are reporting a cybrecrime related to their business.

# Any new packages installed?

> no
# Required followup work

> no
# Checklist:

- [ ] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [ ] I have added and needed tests for my changes (in particular for new screens)
- [ ] I have added a comment to any confusing code
- [ ] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/forms/BusinessInfoForm.js [hunk 1]
```diff
@@ -1,60 +1,86 @@
...
+            as="form"
+            onSubmit={handleSubmit}
+            shouldWrapChildren
+            spacing={6}
+          >
+            <Field
+              name="nameOfBusiness"
+              label={<Trans id="businessPage.nameOfBusiness" />}
+              component={Input}
+            />
```
Comment (amazingphilippe) [inline f2/src/forms/BusinessInfoForm.js:38]: Nice!

File: f2/src/forms/BusinessInfoForm.js [hunk 2]
```diff
@@ -1,60 +1,86 @@
...
+
+            <FormArrayControl
+              name="numberOfEmployee"
+              label={<Trans id="businessPage.numberOfEmployee" />}
+              helperText={<Trans id="businessPage.numberOfEmployeeExample" />}
+            >
+              <RadioAdapter name="numberOfEmployee" value="1 to 99 employees">
+                1 to 99 employees
```
Comment (amazingphilippe) [inline f2/src/forms/BusinessInfoForm.js:60]: We should translate this text, and the other radios! With a <Trans> tag

Comment (kevinyang06) [reply f2/src/forms/BusinessInfoForm.js]: fixed

File: f2/src/locales/fr.json [hunk 3]
```diff
@@ -22,10 +22,13 @@
   "banner.warning": "Attention! Ce site est un prototype. Votre rapport ne sera pas réellement soumis à la GRC.",
   "businessInfoPage.nextButton": "Continuer",
   "businessInfoPage.nextPage": "Étape suivante : Que s’est-il passé?",
-  "businessPage.business": "Comment votre entreprise pourrait-elle être affectée?",
-  "businessPage.businessExample": "Par exemple : actifs, réputation, clients",
+  "businessPage.industry": "Type d'industrie",
```
Comment (amazingphilippe) [inline f2/src/locales/fr.json:25]: Refer to the figma for translated content!
https://www.figma.com/file/R054FzUOrpP13oTbMsQevj/Sprint-9?node-id=3335%3A0

Comment (kevinyang06) [reply f2/src/locales/fr.json]: fixed

File: f2/src/forms/BusinessInfoForm.js [hunk 4]
```diff
@@ -75,6 +75,7 @@ export const BusinessInfoForm = ({ onSubmit }) => {
             <FormArrayControl
               name="numberOfEmployee"
               label={<Trans id="numberOfEmployee.label" />}
+              helperText={<Trans id="numberOfEmployee.labelExample" />}
```
Comment (amazingphilippe) [inline f2/src/forms/BusinessInfoForm.js:78]: Good catch!

**Pull request merged**